SECURITY.md

```diff
@@ -0,0 +1,50 @@
+# Security Policy
+This document outlines security policy and procedures for the CrowdStrike `falconpy` project.
++ [Supported Python versions](#supported-python-versions)
++ [Supported FalconPy versions](#supported-falconpy-versions)
++ [Reporting a potential security vulnerability](#reporting-a-potential-security-vulnerability)
++ [Disclosure and Mitigation Process](#disclosure-and-mitigation-process)
+
+## Supported Python versions
+
+FalconPy functionality is only tested to run under the following versions of Python.
+
+| Version | Supported |
+| :------- | :--------- |
+| 3.9.x   | :white_check_mark: |
+| 3.8.x   | :white_check_mark: |
+| 3.7.x   | :white_check_mark: |
+| 3.6.x   | :white_check_mark: |
+| <= 3.5  | :x: |
+| <= 2.x.x | :x: |
+
+## Supported FalconPy versions
+
+We release patches for security vulnerabilities as they are discovered. 
+
+Version eligibility for receiving patches depends on the CVSS v3.0 Rating:
+
+| CVSS v3.0 | Supported Versions |
+| :------- | :-------------------- |
+| 9.0 - 10.0 | Releases within the past three months |
+| 4.0 - 8.9 | Most recent release |
```

Comment:
Since falconpy is a project, vs product, do we want to take on backporting fixes versus applying them to upstream only?

Reply:
Assuming that we use semantic versioning, I think it only makes sense to backport fixes (security or otherwise) where we've released a new major version (i.e. one with breaking changes). Otherwise, users should be using the most recent released version of the code anyway.

Reply:
This makes sense to me.  I'll move this to be only most recent releases for all reports, CVSS v3.0 4.0 - 10.0.

Reply:
Updated.